services/appveyor/appveyor-build-job.service.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const { renderBuildStatusBadge } = require('../build-status')
+const AppVeyorBase = require('./appveyor-base')
+const { NotFound } = require('..')
+
+module.exports = class AppVeyorBuildJob extends AppVeyorBase {
+  static get route() {
+    return {
+      base: 'appveyor/job/build',
```

Comment:
I was originally planning on making this `appveyor/build/job` but was unable to do so given the badge for the overall build status starts with `appveyor/build/...` and has the trailing branch route param (the route params were being incorrectly interpreted as being part of the branch name)